docs/image-based-upgrade.md

```diff
@@ -472,6 +472,42 @@ This will:
 
 Once completed, the system is ready for the next upgrade.
 
+#### Finalize or Abort failure
+
+If finalize or abort fails, IBU transitions into `FinalizeFailed` or
+`AbortFailed` states respectively.
+
+```yaml
+  status:
+    conditions:
+    - message: failed to cleanup backups. Perform cleanup manually then
+add lca.openshift.io/manualCleanupDone
+        annotation to ibu CR to transition back to Idle
+      observedGeneration: 5
+      reason: AbortFailed
+      status: "False"
+      type: Idle
+```
+
+The condition message indicates which parts of cleanup has failed.
+User should perform the cleanup manually in these states.
+This mostly involves making sure that old stateroot is cleaned up and
+all backup resources are removed.
+Backup resources are `backups.velero.io`, `restores.velero.io` and `deletebackuprequests.velero.io`.
+After manual cleanup user should add 'lca.openshift.io/manualCleanupDone' annotation
```

Comment:
could you link the steps to manual cleanup?

Reply:
I updated the PR. let me know if it makes sense.